.gitignore

```diff
@@ -29,3 +29,8 @@ ocaml/xenopsd/scripts/xen-backend.rules
 ocaml/xenopsd/xentoollog_flags
 
 ocaml/idl/gen_lifecycle.exe
+
+.hugo_build.lock
+go.sum
+
+.DS_Store
```

Comment:
This is macOS specific and should not be here?

Reply:
It is, but some of us use Macs :) It's just to avoid noise.